content/en/docs/Install/kubecf-with-kind.md

```diff
@@ -0,0 +1,172 @@
+# Before you begin
+
+This guideline will provide a quick way to deploy KubeCF with Kind and should be used only for evaluation purposes.
+
+Here is a list of all the tools and versions used when creating these instructions:
+
+```
+> kind version
+kind v0.7.0 go1.13.6 darwin/amd64
+```
+
+```
+❯ bazel version
+Build label: 2.1.0
+Build target: bazel-out/darwin-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
+Build time: Fri Feb 7 13:04:00 2020 (1581080640)
+Build timestamp: 1581080640
+Build timestamp as int: 1581080640
+```
+
+```
+❯ helm version
+version.BuildInfo{Version:"v3.1.1", GitCommit:"afe70585407b420d0097d07b21c47dc511525ac8", GitTreeState:"clean", GoVersion:"go1.13.8"}
+```
+
+```
+❯ docker version
+Client: Docker Engine - Community
+ Version:           19.03.5
+ API version:       1.40
+ Go version:        go1.12.12
+ Git commit:        633a0ea
+ Built:             Wed Nov 13 07:22:34 2019
+ OS/Arch:           darwin/amd64
+ Experimental:      false
+```
+
+```
+❯ kubectl version
+Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-13T18:08:14Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"darwin/amd64"}
+```
+
+```
+❯ cf version
+cf version 6.46.1+4934877ec.2019-08-23z
+```
+
+## Installing Kind
+
+To install Kind please follow the official instructions [here](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+# Create a Kind cluster
+
+To make sure that the cluster is set correctly to host KubeCF, we recommended to go through bazel from the root of the repository:
+
+``` 
+> bazel run //dev/kind:start
```

Comment:
Maybe it's better to not promote the bazel devel target. It is mostly meant for developing, and documentating the canonical way of bringing up the cluster with kind adds value as we can still refer to official docs:

```suggestion
> kind create cluster --name kubecf
```